Description: Adds missing files
 Some files are missing from the final python setup.py install, this patch
 fixes that.
Author: Thomas Goirand <zigo@debian.org>
Forwarded: no
Last-Update: 2015-02-09

Index: python-oslo.middleware/MANIFEST.in
===================================================================
--- python-oslo.middleware.orig/MANIFEST.in
+++ python-oslo.middleware/MANIFEST.in
@@ -6,4 +6,7 @@ exclude .gitreview
 recursive-include oslo/middleware *.py
 recursive-include oslo_middleware *.py
 
+recursive-include oslo/middleware *.py
+recursive-include oslo_middleware *.py
+
 global-exclude *.pyc
\ No newline at end of file
